### DISCUSSION TITLE
Install problem

### DISCUSSION THREAD

**Discussion by komir2004** [akeeba/panopticon#250] [Q&A (Community Support)] [answered]
Hi when try to install Panopticon get message 
`Could not connect to MySQL.`
but SQL data are correct
Database server host name: localhost
when use
Database server host name: 127.0.0.1
install is successful
but when try to login get message 
`mysqli object is not fully initialized`
Thanks for any help

**Comment by nikosdion** [comment]
`localhost` and `127.0.0.1` mean two _very_ different things to MySQL.

`localhost` tries to connect to the database server using UNIX sockets or Windows named pipes, depending on the platform. `127.0.0.1` uses TCP/IP networking. Reference: https://dev.mysql.com/doc/refman/8.0/en/transport-protocols.html#transport-protocol-localhost as well as https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_skip_networking and https://en.wikipedia.org/wiki/Localhost#Special_cases

The `config.php` file you sent me has `public $dbhost = 'localhost';` which your installation cannot use. Therefore you need to edit it and set `public $dbhost = '127.0.0.1';`

**Reply by nikosdion** [reply, marked as answer]
Moreover, I see in the screenshot you provided AND your log file that you enabled “Connect with an SSL certificate”. You did not read the help text, did you? You obviously don't need this and cannot use this. Therefore you need to edit the `config.php` file and set `public $dbencryption = '0';`

Remember, using an SSL certificate **INSTEAD OF** a username and password is something exceedingly rare. You really need to go out of your way to configure a database server like that. If you have no idea what it is, don't use it. That's why the help text explicitly tells you to not use it if you don't know what it means.
